Lib/Inc/CCmdBuffer.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file CCmdBuffer.h
```

Comment:
I don't really understand why this is called CCmdBuffer. There is nothing in the code that makes it command specific.